draft-davidben-tls-trust-expr.md

```diff
@@ -770,7 +770,9 @@ The negotiation mechanism described in this document is analogous to the `certif
 
 Thus, this mechanism SHOULD NOT be used to advertise trust anchors or distrusts which are unique to an individual user. Rather, trust expressions SHOULD be computed based only on the trust anchors common to the relying party's anonymity set ({{Section 3.3 of !RFC6973}}). Additionally, multiple trust expressions may evaluate to the same trust anchor list, so relying parties in the same anonymity set SHOULD send the same trust expression. To achieve this, trust expressions SHOULD be assembled by the root program and configured in relying parties alongside trust store updates.
 
-For example, a web browser may support both a common set of trust anchors configured by the browser vendor, along with user-specified additions and removals. The common trust anchors would reveal, at most, which browser is used, while the user-specified modifications may reveal identifying information about the user. The trust expression SHOULD reflect only the common trust anchors. This limits the benefits of trust anchor agility in two ways:
+For example, a web browser may support both a common set of trust anchors configured by a root program, along with user-specified additions and removals. Applying the above guidance with a typical browser's desired anonymity set, the trust expression would reflect only the common trust anchors for any web browsers using the same trust expression. User-specified modifications may reveal identifying information about the user.
```

Comment:
> the trust expression would reflect only the common trust anchors for any web browsers using the same trust expression

I think this is tautological and doesn't what we're trying to say. Even if, say, a web browser decided to use the user-specified ones, it would be true that the trust expression only reflects trust anchors common to web browsers that use that trust expression. It's just that there's only one of them.

"common trust anchors" was meant in reference to the first sentence in the paragraph:

> For example, a web browser may support both a common set of trust anchors configured by a root program, along with user-specified additions and removals

Reply:
I was tryin to sneak in there the notion that it's not just the anonymity set of a single web browser, multiple web browsers could use the same trust expression provided by a root program.

Reply:
Or at least make that somewhat clearer to the reader, on first cut it sounds like the anonymity set is only the browser unless you are very careful reading it

Reply:
How about this?

```suggestion
For example, a web browser may support both a common set of trust anchors configured by a root program, along with user-specified additions and removals. Applying the above guidance with a typical browser's desired anonymity set, the trust expression would reflect only the common trust anchors and not the user-specified modifications. The user-specified modifications may reveal identifying information about the user, while the common trust anchors would reveal at most, e.g., which version of which browser is used, or even less information if multiple browsers share a root program or the trust anchors were unchanged across multiple versions of the browser.
```